lang/src/org/partiql/lang/eval/CompileOptions.kt

```diff
@@ -41,6 +40,12 @@ enum class ProjectionIterationBehavior {
     FILTER_MISSING, UNFILTERED
 }
 
+enum class StrictnessMode {
+    STANDARD,
+    PERMISSIVE,
+    //TODO: STRICT
+}
```

Comment:
Really, this is `ErrorMode` more than `StrictnessMode`.  And what is the difference between `STANDARD` and `PERMISSIVE` (needs API documentation to indicate intent).  It seems to me we only have `PERMISSIVE` right now...

Reply:
`STANDARD` will be the error behaviour that we currently have.  This is intended for existing customers and will be the default setting. `STRICT` and `PERMISSIVE` are the modes outlined in the specification..